var/spack/repos/builtin/packages/py-markdown-it-py/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyMarkdownItPy(PythonPackage):
+    """Markdown parser, done right.
+       100% CommonMark support, extensions, syntax plugins & high speed """
+
+    homepage = "https://github.com/executablebooks/markdown-it-py"
+    git      = "https://github.com/executablebooks/markdown-it-py"
+    pypi     = "markdown-it-py/markdown-it-py-1.1.0.tar.gz"
+
+    version('1.1.0', sha256='36be6bb3ad987bfdb839f5ba78ddf094552ca38ccbd784ae4f74a4e1419fc6e3')
+
+    depends_on('python@3.6:',   type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@3.6:3',   type=('build', 'run'))
```
`~=` is different than `>=`

Reply:
Planning for python 4 already? That's futureproof :)